.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,20 @@
 <!-- Provide a general summary of your changes in the Title above -->
 
 ## Description
-<!-- Describe your changes in detail -->
+<!-- Describe your changes in detail. -->
+<!-- Please indicate if the changes may break existing functionality. -->
 
 ## Motivation and Context
 <!-- Why is this change required? What problem does it solve? -->
 <!-- If it fixes an open issue, please link to the issue here. -->
 
-## Types of Changes
-<!-- Please select all items that apply either now or after creating the pull request: -->
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Breaking change<sup>1</sup>
-
-<sup>1</sup>The change breaks (or has the potential to break) existing functionality.
 
 ## Checklist:
-<!-- Please select all items that apply either now or after creating the pull request. -->
+<!-- This checklist must be complete before merging the pull request. -->
 <!-- If you are unsure about any of these items, do not hesitate to ask! -->
-- [ ] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
-- [ ] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
+- [ ] I am familiar with the [Contributing Guidelines](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
+- [ ] I agree with the terms of the [Contributor Agreement](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
-- [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
-- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
-- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac/blob/master/changelog.txt) and added all related issue and pull request numbers for future reference (if applicable). See example below.
-
-<!-- Example for a changelog entry: `Fix issue with launching rockets to the moon (#101, #212).` -->
+- [ ] The changes introduced by this pull request are covered by existing or newly introduced **tests**.
```

Comment:
It's weird to have _one_ bold word in the entire document. I agreed with the change to remove bold font from the other linked words. Let's not bold this.
```suggestion
- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
```